src/Functions/csvToJsonString.cpp

```diff
@@ -0,0 +1,408 @@
+#include <DataTypes/DataTypeNullable.h>
```

Comment:
Super Super NIT
Usually, what I do is I put the "" includes before the <> includes.

Why??? 
For compilation performance reasons, the IO is one of the slowness part of the compilation, so I try to reduce as much as possible the includes,
It's very frequent when we include a "" , we also include a <>  (not the other way around) 

so, a trick that I do is I unclude the "" and then the <> to have the least included files.